docs/node/RPC Category/16-loan.md

```diff
@@ -152,7 +152,12 @@ List collateral tokens.
 
 ```ts title="client.loan.listCollateralTokens()"
 interface loan {
-  listCollateralTokens (): Promise<CollateralTokensData>
+  listCollateralTokens (collateralToken: ListCollateralTokens = {}): Promise<CollateralTokensData>
```

Comment:
I am unsure when they will fix this.  They promise me that listCollateralTokens won't accept token as parameter anymore.